qiita_pet/handlers/admin_processing_job.py

```diff
@@ -91,3 +92,40 @@ def get(self):
 
         # return the json in compact form to save transmit size
         self.write(dumps(results, separators=(',', ':')))
+
+
+class SampleValidation(AdminProcessingJobBaseClass):
+    @coroutine
+    @execute_as_transaction
+    def get(self):
+        self._check_access()
+
+        self.render("sample_validation.html", input=True)
+
+    @execute_as_transaction
+    def post(self):
+
+        # Get user-inputted qiita id and sample names
+        qid = self.get_argument("qid")
+        snames = self.get_argument("snames").split()
+
+        qsnames = list(Study(qid).sample_template)
```

Comment:
It's often good to add one maybe two lines of comment in plain English describing what you're doing here. Something like, "Stripping leading Qiita ID from sample-names." would be good. Adding an example of what you expect to see and what you want to transform it into is even better. If something changes in the future that breaks this code, it makes it easier to detect that an expectation was changed.

Reply:
Added comment + example